newrelic/structures_newrelic_alert_policy.go

```diff
@@ -20,6 +20,16 @@ func expandAlertPolicy(d *schema.ResourceData) *alerts.Policy {
 	return &policy
 }
 
+func expandChannelIDs(channelIDs []interface{}) []int {
```

Comment:
Any thoughts on where a better home for this might be? Feels like it could go in `structures_newrelic_alert_channel.go` or in a new `structures_newrelic_alert_policy_channels.go` (which is where I put it in the 1st attempt at this particular feature).

Reply:
Hmm, is this sort of a `util.go` opportunity or something perhaps?

Reply:
I should provide more context - the `structures_*.go` file pattern is a Terraform prescribed pattern that involves "expanding" (usually associated with creation or updating) and "flattening" (usually associated with reads). Some more info can be read [here](https://www.terraform.io/docs/extend/writing-custom-providers.html#implementing-a-more-complex-read). I guess my questions was more around which domain is best for `expandChannelIDs()` - I don't think it should be in `alert_policy`, but can't decide between the two mentioned above :)

Reply:
Since the `structures_newrelic_alert_policy_channels.go` file doesn't exist in this PR, I'm going to put it in `structures_newrelic_alert_channel.go`.